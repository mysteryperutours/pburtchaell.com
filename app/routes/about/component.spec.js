import React from 'react';
import AboutRoute from './component';

describe(AboutRoute.name, () => {
  it('should render component', () => {
    expect(shallow(
      <AboutRoute
        params={{}}
        route={{}}
      />
    )).toMatchSnapshot();
  });
});
