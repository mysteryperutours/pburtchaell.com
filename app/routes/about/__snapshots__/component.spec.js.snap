exports[`AboutRoute should render component 1`] = `
<RouteContainer>
  <Row
    defaultColumn={true}
    elementType="div">
    <Column
      elementType="div"
      size={9}
      style={Object {}}>
      <Text
        type="p">
        Currently, Patrick attends Loyola University New Orleans studying a Bachelor of Science in Design with Honors and expects to graduate in 2018.
      </Text>
      <Text
        type="p">
        Patrick has worked with startups around the world as a  software developer and designer. He is a cofounder and partner at Lawn Chair, a design and software development studio he founded in 2015.
      </Text>
    </Column>
  </Row>
  <Row
    defaultColumn={true}
    elementType="div">
    <Column
      elementType="div"
      size={6}
      style={Object {}}>
      <SocialLinks />
    </Column>
  </Row>
</RouteContainer>
`;
